test/queryselector.spec.ts

```diff
@@ -312,4 +359,100 @@ describe('querySelector', function () {
       expect(second).toEqual([]);
     });
   });
+
```

Comment:
The tests are copied from above. What do you think about this approach? We could avoid this by setting up a loop and try each test for both handlers, but I feel like that will hurt readability of the tests.
We could also cut down the number of tests for this alternative interface.

Reply:
Not a fan of `for` loops around tests — they make it harder to debug. I like the current approach. Consider adding a comment to the tests at the top to indicate that they’re copied below? (You already added a comment in the new place.)

Reply:
I agree. I'll add a comment above as well. Thanks!